docs/book/features/volume_expansion.md

```diff
@@ -1,120 +1,97 @@
 
-CSI Volume Expansion was introduced as an alpha feature in Kubernetes 1.14 and it was promoted to beta in Kubernetes 1.16. The Vanilla vSphere CSI driver supports offline volume expansion which allows a volume to be extended when it is not attached to a node. Currently the support is available only for block volumes.
+CSI Volume Expansion was introduced as an alpha feature in Kubernetes 1.14 and it was promoted to beta in Kubernetes 1.16. The vSphere CSI driver currently extends this support for offline block volumes only i.e allows a block volume to be extended when it is not attached to a node. Check the [supported features](../supported_features_matrix.md) section to verify if your environment conforms to all the required versions. Note that offline volume expansion is available from vSphere CSI v2.0 onwards in Vanilla Kubernetes and v2.1 onwards in Tanzu Kubernetes Grid (TKG) service. 
 
 ## Feature Gate
 
-Expand CSI Volumes feature is promoted to beta in 1.16 so it is enabled by default. For Kubernetes release before 1.16, ExpandCSIVolumes feature gate needs to be enabled for this feature to support volume expansion in CSI drivers.
+Expand CSI Volumes feature was promoted to beta in kubernetes 1.16, therefore it is enabled by default. For Kubernetes releases before 1.16, ExpandCSIVolumes feature gate needs to be enabled for this feature to support volume expansion in CSI drivers.
 
 ## Sidecar Container
 
-An external-resizer sidecar container implements the logic for watching the Kubernetes API for Persistent Volume claim edits and issuing ControllerExpandVolume RPC call against a CSI endpoint and updating PersistentVolume object to reflect new size.
+An external-resizer sidecar container implements the logic of watching the Kubernetes API for Persistent Volume claim edits, issuing the ControllerExpandVolume RPC call against a CSI endpoint and updating the PersistentVolume object to reflect the new size. This container has already been deployed for you as a part of the vsphere-csi-controller pod.
 
-The resize sidecar container will be deployed in the vsphere-csi-controller pod.
+## Requirements
+If your environment adheres to the required kubernetes and vSphere CSI driver versions mentioned above, this feature will work as is in a TKG cluster. Proceed to the `Expand PVC Example` section below to use this feature.
 
-If EXPAND_VOLUME node capability is implemented, NodeExpandVolume will be called by Kubelet on the node if NodeExpansionRequired is set to true in ControllerExpandVolumeResponse.
+However, in order to try out this feature using the vanilla kubernetes driver, modify the RBAC rules and the StorageClass definition as mentioned below in your environment.
 
-## RBAC Rules
+### RBAC Rules
 
-The following  RBAC rules need to be added to support volume expansion:
+Add the `patch` privilege to `persistentvolumes` resource and `update`, `patch` privileges to `persistentvolumeclaims/status` resource.
```

Comment:
Not related to this change, but we've already added these rules by default to 7.0 and above YAMLs. Is there a need to ask a customer to add these privileges explicitly? Maybe its better to just specify that these privileges were added to the YAMLs to support volume expansion

Reply:
I think these are still required for vanilla kubernetes environments. I have excluded this section for TKG.

Reply:
https://github.com/kubernetes-sigs/vsphere-csi-driver/blob/master/manifests/latest/vsphere-7.0/vanilla/rbac/vsphere-csi-controller-rbac.yaml#L16

Reply:
@xing-yang Can you comment on why this section was included in the documentation for offline expansion? Can it be removed?

Reply:
Yeah, this can be removed.  I did this when working on it, but it is not needed in the doc as they are already in our yaml.